kf_lib_data_ingest/target_api_plugins/kids_first_dataservice.py

```diff
@@ -488,6 +488,14 @@ def build_entity(cls, record, get_target_id_from_record):
                 record.get(CONCEPT.SAMPLE.TISSUE_TYPE)
                 or record.get(CONCEPT.BIOSPECIMEN.TISSUE_TYPE)
             ),
+            "has_matched_normal_sample": (
+                record.get(CONCEPT.SAMPLE.HAS_MATCHED_NORMAL_SAMPLE)
+                or record.get(CONCEPT.BIOSPECIMEN.HAS_MATCHED_NORMAL_SAMPLE)
+            ),
+            "external_collection_id": (
+                record.get(CONCEPT.SAMPLE.EXTERNAL_COLLECTION_ID)
+                or record.get(CONCEPT.BIOSPECIMEN.EXTERNAL_COLLECTION_ID)
```

Comment:
on the biospecimen this is just the external sample id - is it going to affect anything to add this here?